### ISSUE TITLE
App seems to boot up, but can't access

### ISSUE THREAD

**Issue by ZempTime** [kriasoft/nodejs-api-starter#94]
Hey! Brand new to docker.

Running:
* MacOS High Sierra 10.13.3 (17D47)
* Completely unmodified clone of the latest version of this repo
* Docker version 18.04.0-ce, build 3d479c0
* docker-compose version 1.21.2, build unknown

When I run `docker-compose up`, everything seems to execute successfully, and I arrive at this line:

```
api_1    | Finished 'build' after 1811ms
api_1    | Node.js API server is listening on http://api:8080/
```

But when I try to access it through chrome in my browser - `http://localhost:8080/` - I don't get anything. Not sure what to do next, or where I'm messing up.

**Comment by shadowbrush** [comment]
Maybe do a `docker ps -a` and see if port `8080` is listed there.

**Comment by ZempTime** [comment]
That gives me:
```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```
This seems promising! Not sure why the docker daemon isn't running, and next step for me is to figure out a way for that to happen

**Comment by shadowbrush** [comment]
Is Docker running? Check the Docker whale icon in the MacOS bar.

**Comment by ZempTime** [comment]
I don't think it was. I just completely uninstalled all previous docker things I added via homebrew, then reinstalled them from what look like the most standard means (dl'ing from docker site). Rebuilding now, whale icon is there, fairly confident this will work. 

...

it is! thanks for your prompts @shadowbrush , much appreciated because I probably woulda been stuck without you. Closing this issue!
